tests/web/SNMPCredential.php

```diff
@@ -43,21 +43,18 @@ public function testCreate()
         $this->addToCleanup(\SNMPCredential::class, ['name' => 'thetestuuidtoremove']);
 
         //load snmp credential form
-        $crawler = $this->http_client->request('GET', $this->base_uri . 'front/snmpcredential.form.php');
+        $this->http_client->request('GET', $this->base_uri . 'front/snmpcredential.form.php');
 
         $auth_passphrase = '¡<av€ry$3"cur€p@ssp\'hr@se>!';
         $priv_passphrase = '>>P4ss"ph&ase<<';
-        $crawler = $this->http_client->request(
-            'POST',
-            $this->base_uri . 'front/snmpcredential.form.php',
+        $crawler = $this->http_client->submitForm(
```

Comment:
The form isn't available in the initial request. It is loaded through an AJAX request later.
Need a `$this->http_client->waitFor('form[name=asset_form]');` before here.